modules/cudf/src/series.ts

```diff
@@ -408,36 +479,38 @@ export class AbstractSeries<T extends DataType = any> {
    * Encode the Series values into integer labels.
    *
    *
-   * @param categories The optional Series of values to encode into integers. Defaults to the unique
-   *   elements in this Series.
-   * @param type The optional integer DataType to use for the returned Series. Defaults to Int32.
+   * @param categories The optional Series of values to encode into integers. Defaults to the
+   *   unique elements in this Series.
+   * @param type The optional integer DataType to use for the returned Series. Defaults to
+   *   Int32.
    * @param nullSentinel The optional value used to indicate missing category. Defaults to -1.
-   * @param memoryResource The optional MemoryResource used to allocate the result Column's device
-   *   memory.
-   * @returns A sequence of encoded integer labels with values between `0` and `n-1` categories, and
-   *   `nullSentinel` for any null values
+   * @param memoryResource The optional MemoryResource used to allocate the result Column's
+   *   device memory.
+   * @returns A sequence of encoded integer labels with values between `0` and `n-1`
+   *   categories, and `nullSentinel` for any null values
    */
   encodeLabels<R extends Integral = Int32>(categories: Series<T>         = this.unique(true),
                                            type: R                       = new Int32 as R,
                                            nullSentinel: R['scalarType'] = -1,
                                            memoryResource?: MemoryResource): Series<R> {
     try {
-      // If there is a failure casting to the current dtype, catch the exception and return encoded
-      // labels with all values set to `nullSentinel`, since this means the Column cannot contain
-      // any of the encoded categories.
+      // If there is a failure casting to the current dtype, catch the exception and return
+      // encoded labels with all values set to `nullSentinel`, since this means the Column
+      // cannot contain any of the encoded categories.
       categories = categories.cast(this.type);
     } catch {
       return Series.sequence(
         {type, init: nullSentinel, step: 0, memoryResource, size: this.length});
     }
     //
-    // 1. Join this Series' values with the `categories` Series to determine the index positions
+    // 1. Join this Series' values with the `categories` Series to determine the index
+    // positions
     //    (i.e. `codes`) of the values to keep.
     // 2. Sort the codes by the original value's position in this Series.
     // 3. Replace missing codes with `nullSentinel`.
     //
-    // Note: Written as a single expression so the intermediate memory allocated for the `join` and
-    // `sortValues` calls are GC'd as soon as possible.
+    // Note: Written as a single expression so the intermediate memory allocated for the
+    // `join` and `sortValues` calls are GC'd as soon as possible.
```

Comment:
would be good to figure out what about my config is somehow different and causing these spurious changes

Reply:
I'm pretty sure this has happened to me as well.